Sources/Contentful/ImageOptions.swift

```diff
@@ -347,3 +345,37 @@ private struct ImageParameters {
     static let quality          = "q"
     static let progressiveJPG   = "fl"
 }
+
+
+// Use CGColor instead of UIColor to enable cross-platform compatibility: macOS, iOS, tvOS, watchOS.
+internal extension CGColor {
```

Comment:
Also: I would love to see tests about this...

Reply:
good call ;-)